include/dxc/Test/CompilationResult.h

```diff
@@ -49,7 +49,7 @@ inline HRESULT GetFirstChildFromCursor(IDxcCursor *cursor,
   return hr;
 }
 
-class TrivialDxcUnsavedFile : IDxcUnsavedFile
+class TrivialDxcUnsavedFile final : IDxcUnsavedFile
```

Comment:
We'd like these final keywords removed since they aren't necessary with the explicit destructor calls.